lib/src/line_scanner.dart

```diff
@@ -61,14 +58,14 @@ class LineScanner extends StringScanner {
       }
     } else {
       final newlines = _newlinesIn(string.substring(newPosition, oldPosition));
-      if (_betweenCRLF) newlines.removeLast();
```

Comment:
This line has likely been a bug since this landed. It was introduced during code review.